src/mono/mono/metadata/loader.c

```diff
@@ -630,6 +605,190 @@ find_method (MonoClass *in_class, MonoClass *ic, const char* name, MonoMethodSig
 	return result;
 }
 
+static MonoMethod *
+find_method_simple (MonoClass *klass, const char *name, const char *qname, const char *fqname,
```

Comment:
Please move `find_method_simple`, `find_method_slow`, `find_method_in_class_unsafe_accessor`,  `mono_unsafe_accessor_find_ctor`, and `mono_unsafe_accessor_find_method` to `mono/metadata/unsafe-accessor.c` since they dont' share any code with the existing `find_method` and `find_method_in_class`.

Reply:
Moved.